src/dataCorrectionsTool/incentiveEligibilityTool.js

```diff
@@ -143,25 +146,23 @@ const handlePaymentRoundSelect = (participant) => {
         !selectButton || 
         !dateOfEligibilityInput) return;
 
-    const { paymentRound, baselinePayment, eligiblePayment, yes, no } = fieldMapping; 
+    const { paymentRound, baselinePayment, eligiblePayment, norcPaymentEligibility, no } = fieldMapping; 
 
     for (let option of dropdownPaymentOptions) {
         option.addEventListener('click', async (e) => {
             participantPaymentRound = e.target.dataset.payment;
             if (participantPaymentRound === fieldMapping['baseline'].toString()) {
                 selectButton.textContent = e.target.textContent;
-
                 try {
                     showAnimation();
-                    const isParticipantEligibleIncentiveResponse = await checkParticipantForEligibleIncentive(participant, participantPaymentRound);
+                    const response = await findParticipant(query);
                     hideAnimation();
+                    const participantData = response.data[0];
+                    localStorage.setItem('participant', JSON.stringify(participantData));
 
-                    const { isEligibleForIncentive, participantData } = isParticipantEligibleIncentiveResponse.data; 
-                    const currentParticipantData = isParticipantEligibleIncentiveResponse.data;
-                    localStorage.setItem('participant', JSON.stringify(currentParticipantData.participantData));
-                    
-                    // check if isParticipantEligibleIncentive.isEligibleForIncentive is true AND current participant data has "130371375.266600170.731498909" value to "NO"
-                    isEligibleForIncentiveUpdate = isEligibleForIncentive && (participantData?.[paymentRound]?.[baselinePayment]?.[eligiblePayment] === no);
+                    const isNORCPaymentEligible = participantData?.[paymentRound]?.[baselinePayment]?.[norcPaymentEligibility] === no;
+                    const isIncentiveEligible = participantData?.[paymentRound]?.[baselinePayment]?.[eligiblePayment] === no;
```

Comment:
"yes" or "no"?

Reply:
This should be "no". Participants with either "no" values will have these fields updated to "yes".